test/jasmine/tests/sankey_test.js

```diff
@@ -771,6 +771,27 @@ describe('sankey tests', function() {
             .then(done);
         });
     });
+
+    it('emits a warning if node.pad is too large', function(done) {
```

Comment:
can you also test something about the node & link thicknesses? Either here or in an image...

Reply:
I went ahead and added a baseline in commit 352cc61!